portal/views/home.py

```diff
@@ -189,56 +193,16 @@ def render_signup_form(request):
     return res
 
 
-def compute_teacher_use_captcha(limits, captcha_limit):
-    using_captcha = (limits['ip'][0] > captcha_limit or limits['email'][0] > captcha_limit)
-    return using_captcha
-
-
 def compute_teacher_should_use_captcha(limits, captcha_limit):
-    should_use_captcha = (limits['ip'][0] >= captcha_limit or limits['email'][0] >= captcha_limit)
+    should_use_captcha = (limits['ip'][0] >= captcha_limit or limits['email'][0] >= captcha_limit) and captcha.CAPTCHA_ENABLED
```

Comment:
Should this be inside the home view?

Reply:
To be honest it was there before so I assumed it was meant to be there. I think it makes sense though since it's the views here that are being called each time per request.